lib/dfe/reference_data/degrees/grades.rb

```diff
@@ -26,66 +28,77 @@ module Degrees
           'e2fe18d4-8655-47cf-ab1a-8c3e0b0f078f' =>
           { hesa_code: '02',
             name: 'Upper second-class honours (2:1)',
+            abbreviation: '2:1',
```

Comment:
Agree with Ed's points, can this also be changed to 'Upper second' so it's consistent with the other abbreviations